syntax = "proto3";

package schemas;

message Alert {
  string alert_id = 1;
  string created_at = 2; // ISO 8601
  repeated string event_ids = 3;
  int32 threat_score = 4;
  string severity = 5;
  string status = 6;
  string verdict = 7;
  string reasoning = 8; // JSON string
  repeated string affected_assets = 9; // JSON strings or simple strings
}
